packages/core/src/components/Label.tsx

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import { LabelVariant } from "../index";
+import classNames from "classnames";
+
+interface Props {
+    variant?: LabelVariant;
+    forceCompact?: boolean;
+    srOnly?: boolean;
+    standAlone?: boolean;
+    htmlFor?: string;
+}
+
+export const Label: React.FC<Props> = ({ variant = "medium", forceCompact, srOnly, children, standAlone, htmlFor }) => {
+    const className = classNames("jkl-label", {
+        [`jkl-label--${variant}`]: variant,
+        "jkl-label--compact": forceCompact,
+        "jkl-label--sr-only": srOnly,
+    });
+
+    const C = standAlone ? "label" : "span";
+
+    if (!standAlone && htmlFor) {
+        htmlFor = undefined;
+        if (process.env.NODE_ENV !== "production") {
+            console.warn(
```

Comment:
Er det mulig å bruke typescript til å vise at `htmlFor` er avhengig av at `standalone` må være truthy?

Reply:
Vet ikke helt 🤔 Lurer for øvrig på om vi burde snu om på hele mønsteret her. Nesten alle komponentene som bruker denne er nå skrevet om til å bruke `standAlone`-varianten. De eneste jeg vet om som bruker `Label` utenfor Jøkul er AHV, og de bruker nok også `standAlone`-varianten..